src/batched/KokkosBatched_Axpy_Decl.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef __KOKKOSBATCHED_AXPY_DECL_HPP__
+#define __KOKKOSBATCHED_AXPY_DECL_HPP__
+
+
```

Comment:
Should it be src/batched/sparse ?

What about the namespace? Is KokkosBatched the right level?